### PR TITLE
add log analysis for ce

### PR DESCRIPTION
**Pull request by zjjlivein**
### PR Category

Execute Infrastructure

### PR Types

Improvements

### Description

add AI log analysis for ce

### 是否引起精度变化

否

### CODE REVIEW

File: ci/check_ce_precision.sh [hunk 1]
```diff
@@ -14,19 +14,39 @@
...
 fi
 case_gt_file=${base_name}_${case_name}_gt.txt
+
+if [[ ! -f "${case_name}.txt" && "$case_name" == *glm* ]]; then
+    cp glm45_fleet/${case_name}.txt ./
+fi
+
+if [[ "$update_baseline" == "true" ]]; then
+    cp ${case_name}.txt ${case_gt_file}
+    echo "Update baseline: ${case_gt_file}"
```
Comment (risemeup1111) [inline ci/check_ce_precision.sh:30]: P1 **优先级：P1**

这里直接把 `${case_name}.txt` 复制成 `${case_gt_file}` 会把完整训练日志上传为基线；但后续同脚本用 `check_loss.py --gt_file` 读取这个文件，而 `parse_ground_truth()` 期望基线每行是 `step loss` 两列数字。更新一次基线后，后续 CE 精度检查会在解析 GT 或匹配 common steps 时失败/失效。请在上传前先从日志中抽取 `global_step` 和 `loss`，生成与现有下载基线一致的两列格式，例如：

```suggestion
    python - "${case_name}.txt" "${case_gt_file}" <<'PY'
import re
import sys

src, dst = sys.argv[1], sys.argv[2]
step_pattern = re.compile(r"global_step:\s*(\d+)")
loss_pattern = re.compile(r"loss:\s*([0-9.]+)")
rows = []
with open(src, encoding="utf-8", errors="ignore") as f:
    for line in f:
        step = step_pattern.search(line)
        loss = loss_pattern.search(line)
        if step and loss:
            rows.append(f"{step.group(1)} {loss.group(1)}\n")
if not rows:
    raise SystemExit(f"Failed to extract ground truth from {src}")
with open(dst, "w") as f:
    f.writelines(rows)
PY
    echo "Update baseline: ${case_gt_file}"
```

**Review by risemeup1111 — CHANGES_REQUESTED** [review]
已复查新提交：`yamlfmt` 格式问题和 PR 模板字段已补齐。当前仍有一个会影响 CE baseline 更新正确性的阻塞问题，细节已放在行级评论里；`Check approval` 失败日志显示为 runner 访问 GitHub 超时，未作为本次代码问题处理。

<div align="right">
   <sup>Powered by Nyanpasu with gpt-5.5 xhigh, please check the suggestions carefully.</sup>
</div>

**Review by tianlef — APPROVED** [review]
